active_directory_ldap/komand_active_directory_ldap/actions/modify_groups/action.py

```diff
@@ -24,14 +24,17 @@ def run(self, params={}):
         add_remove = params.get('add_remove')
 
         # Normalize dn
-        dn = formatter.format_dn(dn)[0]
+        dn, search_base = formatter.format_dn(dn)
         dn = formatter.unescape_asterisk(dn)
         self.logger.info(f'Escaped DN {dn}')
         # Normalize group dn
         group_dn = formatter.format_dn(group_dn)[0]
         group_dn = formatter.unescape_asterisk(group_dn)
         self.logger.info(f'Escaped group DN {group_dn}')
 
+        # Check that dn exists in AD
+        self.check_that_user_dn_is_valid(conn, dn, search_base)
+
         if add_remove == 'add':
             try:
                 group = extend.ad_add_members_to_groups(conn, dn, group_dn)
```

Comment:
We should pass the fix=True flag to the ad_add_members_to_groups and it should throw the invalid group error that we're catching.

Reply:
I added fix=True to `ad_add_members_to_groups`. I also added `raise_error=True` in both: `ad_add_members_to_groups` and `ad_remove_members_from_groups`. This allow us to catch this error and show exactly error to user with `PluginException`